src/themes/onvote/index.ts

```diff
@@ -34,6 +34,10 @@ export const theme = extendTheme(vtheme, {
       '.brand-theme': {
         fontFamily: 'pixeloidsans, monospace',
       },
+      '.calculator': {
+        border: '1px solid',
+        borderColor: 'calculator.border',
+      },
```

Comment:
You should avoid using classes unless you don't have other options. In this case you should have defined a custom style object, like with any other custom component: https://v2.chakra-ui.com/docs/styled-system/component-style#styling-single-part-components